src/tree/builder/htmlClasses.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * All HTML element tags that we treat as formatting.
+ * @type {string[]}
+ * @const
+ *
+ * @memberOf module:tree/builder
+ */
+const formattingElements = [ "a", "abbr", "audio", "b", "bdo", "br", "button", "canvas", "cite", "code", "command",
+	"data", "datalist", "dfn", "del", "em", "embed", "i", "iframe", "img", "input", "kbd", "keygen", "label", "mark",
+	"math", "meter", "noscript", "object", "output", "progress", "q", "ruby", "samp", "script", "select", "small",
```

Comment:
I am using [this list on HTML phrasing content on Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content).

I think having a structure like this is valid HTML according to MDN:
```html
<p>This is some script <script>console.log("logging something");</script></p>
```
So I think it should be added as formatting to the paragraph for completeness.

I think it would not hurt for me to add a `@see` JSDoc tag to point towards the MDN list I used, for documentation purposes.